CHANGES.md

```diff
@@ -78,7 +78,8 @@ OpenSSL 3.0
    *Tomáš Mráz*
 
  * Removed RSA padding mode for SSLv23 (which was only used for
-   SSLv2)
+   SSLv2). This include the functions RSA_padding_check_SSLv23() and
```

Comment:
Should it be rather `includes`? And please use backticks around the `-ssl` and `rsautl`.

Reply:
updated commit pushed.